music21/meter/base.py

```diff
@@ -41,6 +41,7 @@
 environLocal = environment.Environment('meter')
 
 if TYPE_CHECKING:
+    from music21 import meter
```

Comment:
Unused

Reply:
True. This was in for a different kind of type checking now removed.